spec/controllers/projects_controller_spec.rb

```diff
@@ -26,11 +27,15 @@
       end
 
       context 'rendering the show' do
-        before :each do
-          post :create, :project => subject_params
-        end
 
+        subject { post :create, :project => subject_params }
```

Comment:
The call to post is better suited into a `before do; end`. Is there a special reason for making it within the subject?